code/modules/xenoarcheaology/artifacts/triggers/gas.dm

```diff
@@ -5,7 +5,7 @@
 
 /datum/artifact_trigger/gas/New()
 	if(!gas_needed)
-		gas_needed = list(pick(subtypesof(/decl/material/gas)) = rand(1,10))
+		gas_needed = list(pick(decls_repository.get_decls_of_subtype(/decl/material/gas)) = rand(1,10))
```

Comment:
This should either be `get_decl_paths_of_subtype()`, or line 9 should be changed to not use GET_DECL.